### DISCUSSION TITLE
Σάββατο ή Κυριακή 26-27 Νοε 6μμ για συμπληρωματική ΟΣΣ2;

### DISCUSSION THREAD

**Discussion by epidrome** [sdy60/syllabus#39] [Polls]
Διαθέσιμες επιλογές για συμπληρωματική ΟΣΣ

**Comment by afoscolou** [comment]
Εννοείτε το Σάββατο 26/11 στις 6μμ ή την Κυριακή 27/11 στις 6μμ; Η οι επιλογές αυτές είναι γι αυτό το ΣΚ όπως λέει το θέμα της συζήτησης;

**Reply by epidrome** [reply]
αρχικά νόμιζα ότι ήταν βιαστικό και ήταν για αυτό το ΣΚ
η αλήθεια είναι ότι είναι λίγο τελευταία στιγμή
από τις ψήφους (έξι, λιγότεροι από τους μισούς δλδ) 
φάνηκε ότι δεν υπήρχε ικανό ενδιαφέρον

ενημερώθηκαν οι ημερομηνίες για το άλλο ΣΚ

**Comment by dimidimots** [comment]
Καλημέρα σας,


τελικά πότε θα κάνουμε το meeting, υπάρχει κάτι επίσημο ;

Ευχαριστώ πολύ.
Δημήτρης.

**Reply by epidrome** [reply]
επίσημα σίγουρα θα γίνει όταν λέει το ΕΑΠ δλδ Σαβ. 10 Δεκ. 4μμ

ανεπίσημα θα κάνουμε συμπληρωματική ΟΣΣ2 σύμφωνα με την παραπάνω ψηφοφορία #44
